.github/workflows/python-publish-aws-lambda-sdk.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         run: |
           cd python/packages
 
-          ./aws-lambda-sdk/scripts/build-layer-archive.sh build extension.internal.zip
+          ./aws-lambda-sdk/scripts/build-layer-archive.sh extension.internal.zip
```

Comment:
To maintain same convention it'll be probably best if same as on Node.js side it's built into `aws-lambda-sdk/dist` folder, and that we also ensure that this folder is in `.gitignore` (as Node.js one is: https://github.com/serverless/console/blob/687ed085e40b003c41aa1fde5012b69e0b887cb4/node/packages/aws-lambda-sdk/.gitignore#LL1C6-L1C6)